### ISSUE TITLE
Disable Icon in the Cast Bar?

### ISSUE THREAD

**Issue by phendrome** [obble/md#5]
Not sure if I'm digging this icon in the cast bar. Feels off. Perhaps it'd look better at the very edge of the cast bar?

I understand that you're trying to save space if you compare it to how you've done it in the normal modui.

![md](https://user-images.githubusercontent.com/2565750/36338392-facdbfb8-13ad-11e8-9a28-2ebe515873b4.png)

Anyway, come on maestro -- would love to see some more development into this interface!

**Comment by obble** [comment]
its definitely not quite * right * yet, i agree

should be back with updates imminently

**Comment by phendrome** [comment]
How would you want to solve it? I like the idea of a cut icon, it looks neat. But making it feel a part of the UI is slightly tougher.

That's awesome -- can't wait! Get a Discord going my dude.

**Comment by phendrome** [comment]
There's no way to contact you other than here, so I'll give it a shot. 

Big fan of the UI -- and would love to see some more progress. Hard to find the time to manage it?

**Comment by obble** [comment]
yeah been busy with other stuff/moving between classic and [retail](https://github.com/obble/iipui). I think im coming back for KIII though, and hoping to make some progress on this and at least publish it as modui.
